pvlib/tests/iotools/test_psm3.py

```diff
@@ -149,13 +153,49 @@ def io_input(request):
 
 def test_parse_psm3(io_input):
     """test parse_psm3"""
-    data, metadata = psm3.parse_psm3(io_input)
+    data, metadata = psm3.parse_psm3(io_input, map_variables=False)
     expected = pd.read_csv(YEAR_TEST_DATA)
     assert_psm3_equal(data, metadata, expected)
 
 
 def test_read_psm3():
     """test read_psm3"""
-    data, metadata = psm3.read_psm3(MANUAL_TEST_DATA)
+    data, metadata = psm3.read_psm3(MANUAL_TEST_DATA, map_variables=False)
     expected = pd.read_csv(YEAR_TEST_DATA)
     assert_psm3_equal(data, metadata, expected)
+
+
+def test_read_psm3_map_variables():
+    """test read_psm3 map_variables=True"""
+    data, metadata = psm3.read_psm3(MANUAL_TEST_DATA, map_variables=True)
+    columns_mapped = ['Year', 'Month', 'Day', 'Hour', 'Minute', 'dhi', 'dni',
+                      'ghi', 'dhi_clear', 'dni_clear', 'ghi_clear',
+                      'Cloud Type', 'Dew Point', 'apparent_zenith',
```

Comment:
@AdamRJensen this test is failing locally for me because of an `apparent_zenith`/`solar_zenith` difference.  Shouldn't this be `solar_zenith`, and shouldn't this test have failed before we merged the PR?  I'm very confused why this wasn't failing before.

Reply:
Yes indeed. Sorry for the slow response. Thanks for the fix!